lnbits/extensions/gerty/helpers.py

```diff
@@ -41,17 +41,17 @@ def get_text_item_dict(
         line_width = 25
 
     # Get font sizes for Gerty mini
-    if gerty_type.lower() == "mini gerty":
-        if font_size <= 12:
-            font_size = 1
-        if font_size <= 15:
-            font_size = 1
-        elif font_size <= 20:
-            font_size = 2
-        elif font_size <= 40:
-            font_size = 2
-        else:
-            font_size = 5
+    # if gerty_type.lower() == "mini gerty":
```

Comment:
- dead code should be removed (if not planned to be re-enabled in the near future)